src/components/pages/Login/LoginContainer.js

```diff
@@ -1,55 +1,14 @@
-import React, { useEffect } from 'react';
```

Comment:
Nice job implementing the redirect functionality in this component! Since it is only 57 lines that were removed (maybe that is a lot) I am not too worried about it being not atomic in nature.

Reply:
Thank you @kdneldor, the Okta Widget was pretty heavy on code but Auth0 is very lightweight and only requires a few lines to get the job done :)